src/components/site/linearProgress/index.jsx

```diff
@@ -12,6 +13,10 @@ const LinearProgress = ({ min, max, value, id, labelledById }) => (
       aria-labelledby={labelledById}
       style={{ width: `${value}%` }}
     ></div>
+    <LinearProgressMarker completed hoverable progress={value}>
+      <span className="progress-linear-hover-text">{`${value}%`}</span>
+    </LinearProgressMarker>
+    {children}
```

Comment:
let children be put into the linear progress component for the purpose of displaying checkpoint and finish point markers.